integration.md

```diff
@@ -81,30 +81,32 @@ We've previously stated that the forms presented by Klarna are eventually posted
 ```ruby
 # Handle POST requests to '/purchase'
 post '/purchase' do
+  @user = User.create(session, params[:userToken], 'user@email.com') unless @user
   purchase = Klarna::Purchase.new(
-    user_token:    params[:userToken],
-    reference:     "IA-#{params[:article_id]}",
-    name:          'Interesting Article',
+    user_token:    @user.token,
+    reference:     'subscription',
+    name:          'Monthly subscription',
     amount:        99,
     tax:           6,
     origin_proof:  params[:origin_proof]
   )
 
-  authorization_response = purchase.authorize!
-  if authorization_response.success?
+  success, response_data, response_code = purchase.order!
+  if success
+    @user.subscribe!
     article = Article.find(params[:article_id])
-    return json data: article.paid_content, klarna_response: authorization_response.data
+    return json data: article.paid_content, klarna_response: response_data
   else
-    status 500
-    return json klarna_response: authorization_response.data
+    status response_code and return json data: {}, klarna_response: response_data
   end
 end
+
 ```
 
 Let us go over the above code and see what it does.
 First, a new purchase is created using the information posted with the form. Should you care to delve into the specifics of the `Klarna::Purchase` object you are welcome to [look at the source code](https://github.com/klarna/sample-digital-goods-backend/blob/master/models/klarna/purchase.rb), but simply put it is a wrapper object for [Klarna's on Demand API](http://docs.inapp.apiary.io/). Specifically, for the creation of [orders](http://docs.inapp.apiary.io/#orders).
 
-Once created, we attempt to authorize the purchase. If everything goes smoothly, we will then respond with the article's paid contents and purchase info. Otherwise, we will return an indicative error.
+Once created, we attempt to make an order for the purchase. If everything goes smoothly, we will then respond with the article's paid contents and purchase info. Otherwise, we will return an indicative error.
```

Comment:
Make an order for the purchase? That does not compute. Would we like to change this to `Once created, we attempt to place an order`?